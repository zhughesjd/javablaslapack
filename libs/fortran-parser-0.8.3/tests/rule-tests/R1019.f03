! Test char-string-edit-desc
!      char-string-edit-desc  is  char-literal-constant
010 format('(10f8.2)')

end


