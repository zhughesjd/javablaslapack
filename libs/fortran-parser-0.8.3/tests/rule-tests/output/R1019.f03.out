*******************************************
args[1]: R1019.f03
R1101:main-program__begin:
R 313:label: lbl=[@4,85:87='010',<10>,3:0]
R1003:format-item-list__begin:
R1003:format-item: descOrDigit=[@-1,0:0=''(10f8.2)'',<196>,3:11] hasFormatItemList=false
R1003:format-item-list: count=1
R1002:format-specification: hasFormatItemList=true
R1001:format-stmt: label=[@4,85:87='010',<10>,3:0]
R 207:declaration-construct:
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=1
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
Parser exiting normally
********************************************
